app/services/UIBuilder.scala

```diff
@@ -196,4 +197,14 @@ class UIBuilder {
   private def fromSectionAndNoteCallout(caption: Text, nc: NoteCallout)(implicit stanzaIdToUrlMap: Map[String, String], lang: Lang): UIComponent =
     Details(caption, Seq(TextBuilder.fromPhrase(nc.text)))
 
+  private def fromSequence(sequence: Sequence, components: Seq[UIComponent])
+                          (implicit stanzaIdToUrlMap: Map[String, String], lang: Lang): UIComponent = {
+    val (errMsgs, uiElements) = partitionComponents(components, Seq.empty, Seq.empty)
+
+    val (text, hint) = TextBuilder.singleTextWithOptionalHint(sequence.text)
```

Comment:
There is a bug in Question (in the backlog) to replace use of singleTextWithOptionalHint with something like fromPhrase which also extracts the hint. Inputs dont have this bug, it prevents what becomes the H1 from including label references (this is one of Abby's discoveries)

Reply:
I've updated task EG-1307 which records this defect such that sequence should also be fixed.